src/Microsoft.DotNet.XHarness.CLI/Commands/iOS/iOSTestCommand.cs

```diff
@@ -78,7 +78,8 @@ protected override async Task<ExitCode> InvokeInternal()
             var processManager = new ProcessManager(_arguments.XcodeRoot, _arguments.MlaunchPath);
             var deviceLoader = new HardwareDeviceLoader(processManager);
             var simulatorLoader = new SimulatorLoader(processManager);
-            var tunnelBore = (_channel == CommunicationChannel.UsbTunnel) ? new TunnelBore(processManager) : null;
+            var hasSimulators = _arguments.TestTargets.Where(t => t.IsSimulator()).Any();
```

Comment:
Can you please decide this lower in the foreach and do tunnels for devices and network for sims?